app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -0,0 +1,107 @@
+package org.mifos.selfserviceapp.ui.activities;
+
+
+import android.os.Bundle;
+import android.support.design.widget.NavigationView;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+
+import android.view.MenuItem;
+import android.view.View;
+import android.support.v7.widget.Toolbar;
+
+import org.mifos.selfserviceapp.R;
+
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * @author Vishwajeet
+ * @since 14/07/2016
+ */
+
+public class HomeActivity extends BaseActivity implements NavigationView.OnNavigationItemSelectedListener {
+
+    private NavigationView mNavigationView;
+    private DrawerLayout mDrawerLayout;
+
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        getActivityComponent().inject(this);
+
+        setContentView(R.layout.activity_home);
+
+        ButterKnife.bind(this);
+
+        if (toolbar != null) {
+            setSupportActionBar(toolbar);
+            toolbar.setTitle(getString(R.string.home));
+        }
+
+        setupNavigationBar();
+    }
+
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
+        // ignore the current selected item
+        if (item.isChecked()) {
+            mDrawerLayout.closeDrawer(GravityCompat.START);
+            return false;
+        }
+
+        // select which item to open
+        switch (item.getItemId()) {
+            case R.id.item_accounts:
+                break;
+            case R.id.item_funds_transfer:
+                break;
+            case R.id.item_recent_transactions:
+                break;
+            case R.id.item_questionnaire:
+                break;
+            case R.id.item_about_us:
+                break;
+        }
+
+        // close the drawer
+        mDrawerLayout.closeDrawer(GravityCompat.START);
+        mNavigationView.setCheckedItem(R.id.item_accounts);
+        return true;
+    }
+
+    /**
+     *This method is used to set up the navigation drawer for
+     * self-service application
+     */
+    private void setupNavigationBar() {
+
+        // setup navigation view
+        mNavigationView = (NavigationView) findViewById(R.id.navigation_view);
```

Comment:
Why ```findViewById``` here? You already have ButterKnife library! Consider usage of ```@BindView``` for the NagivationView and DrawerLayout.